src/hb-ot-hmtx-table.hh

```diff
@@ -79,6 +79,7 @@ struct hmtxvmtx
     hb_blob_t *src_blob = OT::Sanitizer<H>().sanitize (plan->source->reference_table (T::tableTag));
     if (unlikely(!src_blob))
```

Comment:
Oh. Sanitizer never returns nullptr. It returns the hb_blob_get_empty().

Reply:
Fixed by way of copy writeable